README.md

```diff
@@ -2,4 +2,7 @@
 Scotland Graduation Project - Digital Welcome Pack for refugees
 #To run the server locally
 use npm run dev that it will make the server work automatically when happens any change
-#server port 3001 client port 3000
\ No newline at end of file
+ #To run the server locally 
```

Comment:
for the README to be formatted properly, you need to have an empty line before the header the **#** has to be at the start of the line and it has to have space right after the `# Title`  otherwise it looks like this: https://github.com/kharazzoro/legodi/blob/e5cd45a6066d39b1b6632fe57d80a52533767740/README.md